pyam/iiasa.py

```diff
@@ -330,15 +330,16 @@ def _match(data, patterns):
         # pass empty list to API if all regions selected
         if len(regions) == len(self.regions()):
             regions = []
-
+        logger.debug(f"Prepared filter for {len(regions)} region(s), "
+                     f"{len(variables)} variables and {len(runs)} runs")
         data = {
             "filters": {
                 "regions": list(regions),
                 "variables": list(variables),
                 "runs": list(runs),
                 "years": [],
                 "units": [],
-                "times": []
+                "timeslices": []
```

Comment:
wouldn't it make more sense for this to be called "subannual"?

Reply:
I don't like `subannual` to be name of the filter which contains list of time slices. Same with DB table name. If you think it won't work for you we can change it later on (it will require redeployment of all Scenario Explorer instances).